### PR TITLE
translation(hindi): add security.md

### PR DESCRIPTION
**Pull request by ChiragAgarwal7**
### Affected components 

- Server
- Console
- CLI
- Docs
- Community Content
- Build System
- Tests
- Other (list it)

### CODE REVIEW

**Comment by ChiragAgarwal7** [comment]
sure

File: SECURITY.md [hunk 1]
```diff
@@ -35,3 +35,4 @@ A public disclosure date is negotiated by the Hasura product security team and t
 This document is available in the following translations:
 
 - [French :fr:](translations/SECURITY.french.md)
+- [Hindi :hi:](translations/SECURITY.hindi.md)
```
Comment (marionschleifer) [inline SECURITY.md]: The flag doesn't show here:

image

File: translations/README.md [hunk 2]
```diff
@@ -6,5 +6,5 @@
 | Event Triggers  | [:uk: English](../event-triggers.md)  ￨ [:fr: Français](event-triggers.french.md)  |
 | Remote Schemas  | [:uk: English](../remote-schemas.md)  ￨ [:fr: Français](remote-schemas.french.md)  |
 | Contributing    | [:uk: English](../CONTRIBUTING.md)    ￨ [:fr: Français](CONTRIBUTING.french.md)    |
-| Security        | [:uk: English](../SECURITY.md)        ￨ [:fr: Français](SECURITY.french.md)        |
+| Security        | [:uk: English](../SECURITY.md)        ￨ [:fr: Français](SECURITY.french.md)        | [:hi: हिंदी](SECURITY.hindi.md)        |
```
Comment (marionschleifer) [inline translations/README.md]: Also, something isn't right here:

image

**Comment by marionschleifer** [comment]
@ChiragAgarwal7 are you interested in implementing the suggested changes? 🙂

**Comment by ChiragAgarwal7** [comment]
Sorry i had some problems, i will do it by tomorrow

**Comment by marionschleifer** [comment]
@ChiragAgarwal7 just wanted to ask once more about the required changes in this PR.

**Comment by ChiragAgarwal7** [comment]
Sorry for being so late, I m having trouble showing the flag image as I can't find where the other flag images are stored, can u help me with that? I have corrected all the other issues.

**Comment by marionschleifer** [comment]
@ChiragAgarwal7 sure. You can just copy - paste this flag:  🇮🇳

**Comment by ChiragAgarwal7** [comment]
I have made the changes you can merge it if its fine.

**Pull request merged**